index.html

```diff
@@ -401,14 +400,6 @@ <h2>Dictionary {{RTCRtpEncodingParameters}} Members</h2>
           allows the <a>user agent</a> to adapt its bandwidth allocation strategy
           based on the current network conditions.</p>
         </dd>
-        <dt>
-          <dfn data-idl>codec</dfn> of type <span class="idlMemberType">RTCRtpCodec</span>
-        </dt>
-        <dd>
-          <p>Optional value selecting which codec is used for this encoding's
-            RTP stream. The {{RTCRtpCodec}} dictionary is defined in [[WEBRTC]].
-            If [=map/exists|absent=], the user agent can chose to use any negotiated codec.</p>
-        </dd>
```

Comment:
There's more to https://github.com/w3c/webrtc-extensions/pull/147 that needs to be moved I think.